### PR TITLE
Docs: Upgrade Weblate language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
This became the topic of not wanting or daring to upgrade, and I think this will help a little bit.

### CODE REVIEW

File: docs/admin/upgrade.rst [hunk 1]
```diff
@@ -71,164 +70,164 @@ work, but is not as well tested as single version upgrades.
...
+Upgrading from 4.2 to 4.3
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Please follow :ref:`generic-upgrade-instructions` in order to perform update.
+Please follow :ref:`generic-upgrade-instructions` to upgrade.
 
 Notable configuration or dependencies changes:
 
-* There are some changes in quality checks, you might want to include them in case you modified the :setting:`CHECK_LIST`.
-* The source language attribute was moved from project to a component what is exposed in the API. You will need to update :ref:`wlc` in case you are using it.
```
Comment (comradekingu) [inline docs/admin/upgrade.rst:159]: what is?

File: docs/admin/upgrade.rst [hunk 2]
```diff
@@ -71,164 +70,164 @@ work, but is not as well tested as single version upgrades.
...
 * Django 3.1 is now required.
-* In case you are using MySQL or MariaDB, the minimal required versions have increased, see :ref:`mysql`.
+* If you are using MySQL or MariaDB, the minimal required versions increased, see :ref:`mysql`.
 
 .. versionchanged:: 4.4.1
 
-   * :ref:`mono_gettext` now uses both ``msgid`` and ``msgctxt`` when present. This will change identification of translation strings in such files breaking links to Weblate extended data such as screenshots or review states. Please make sure you commit pending changes in such files prior upgrading and it is recommeded to force loading of affected component using :djadmin:`loadpo`.
-   * Increased minimal required version of translate-toolkit to address several file format issues.
+   * :ref:`mono_gettext` now uses both ``msgid`` and ``msgctxt`` when present. This changes identification of translation strings in such files, breaking links to extended Weblate data such as screenshots or review states. Please ensure you commit pending changes in such files prior to upgrading. It is recommeded to force-load affected components using :djadmin:`loadpo`.
```
Comment (comradekingu) [inline docs/admin/upgrade.rst:187]:
```suggestion
   * :ref:`mono_gettext` now uses both ``msgid`` and ``msgctxt`` when present. This changes identification of translation strings in such files, breaking links to extended Weblate data such as screenshots or review states. Please ensure you commit pending changes in such files prior to upgrading. It is recommended to force-load affected components using :djadmin:`loadpo`.
```

File: docs/admin/upgrade.rst [hunk 3]
```diff
@@ -71,164 +70,164 @@ work, but is not as well tested as single version upgrades.
...
 
-Please follow :ref:`generic-upgrade-instructions` in order to perform update.
+Please follow :ref:`generic-upgrade-instructions` to upgrade.
 
 Notable configuration or dependencies changes:
 
-* There are new file formats, you might want to include them in case you modified the :setting:`WEBLATE_FORMATS`.
+* New file formats. You might want to include them in case you modified the :setting:`WEBLATE_FORMATS`.
 * API for creating components now automatically uses :ref:`internal-urls`, see :http:post:`/api/projects/(string:project)/components/`.
 * There is a change in dependencies and :setting:`django:PASSWORD_HASHERS` to prefer Argon2 for passwords hashing.
```
Comment (comradekingu) [inline docs/admin/upgrade.rst:219]: A dependency change of what? Now prefers? "password hashing"

File: docs/admin/upgrade.rst [hunk 4]
```diff
@@ -240,9 +239,9 @@ Notable configuration or dependencies changes:
 Upgrading from Python 2 to Python 3
 -----------------------------------
 
-Weblate no longer supports Python older than 3.5. In case you are still running
-on older version, please perform migration to Python 3 first on existing
-version and upgrade later. See `Upgrading from Python 2 to Python 3 in the Weblate
+Weblate no longer supports Python older than 3.5. If you are still running
+on older version, please migrate to Python 3 first on an existing Weblate
```
Comment (comradekingu) [inline docs/admin/upgrade.rst:243]: Just making 100% sure this shouldn't be "Python version"

File: docs/admin/upgrade.rst [hunk 5]
```diff
@@ -331,17 +331,17 @@ The simplest approach for migration is to utilize Django JSON dumps. This works
...
-3. Dump legacy database and import to PostgreSQL
+3. Dump the legacy database and import it in PostgreSQL
 
 .. code-block:: sh
 
    weblate dumpdata --all --output weblate.json
    weblate loaddata weblate.json --database=postgresql
 
-4. Adjust :setting:`django:DATABASES` to use just PostgreSQL database as default,
+4. Adjust :setting:`django:DATABASES` to only use the PostgreSQL database,
```
Comment (comradekingu) [inline docs/admin/upgrade.rst:341]: Not 100% sure there isn't a fallback database.

File: docs/admin/upgrade.rst [hunk 6]
```diff
@@ -350,7 +350,7 @@ Migrating to PostgreSQL using pgloader
 
 The `pgloader`_ is a generic migration tool to migrate data to PostgreSQL. You can use it to migrate Weblate database.
 
-1. Adjust your :file:`settings.py` to use PostgreSQL as a database.
+1. Adjust your :file:`settings.py` to use PostgreSQL as your database.
```
Comment (comradekingu) [inline docs/admin/upgrade.rst:353]: Are multiple databases at once possible? Otherwise this is good.

File: docs/admin/upgrade.rst [hunk 7]
```diff
@@ -71,164 +70,164 @@ work, but is not as well tested as single version upgrades.
...
 * Django 3.1 is now required.
-* In case you are using MySQL or MariaDB, the minimal required versions have increased, see :ref:`mysql`.
+* If you are using MySQL or MariaDB, the minimal required versions increased, see :ref:`mysql`.
 
 .. versionchanged:: 4.4.1
 
-   * :ref:`mono_gettext` now uses both ``msgid`` and ``msgctxt`` when present. This will change identification of translation strings in such files breaking links to Weblate extended data such as screenshots or review states. Please make sure you commit pending changes in such files prior upgrading and it is recommeded to force loading of affected component using :djadmin:`loadpo`.
-   * Increased minimal required version of translate-toolkit to address several file format issues.
+   * :ref:`mono_gettext` now uses both ``msgid`` and ``msgctxt`` when present. This changes identification of translation strings in such files, breaking links to extended Weblate data such as screenshots or review states. Please ensure you commit pending changes in such files prior to upgrading. It is recommended to force-load affected components using :djadmin:`loadpo`.
```
Comment (comradekingu) [inline docs/admin/upgrade.rst:187]:
```suggestion
   * :ref:`mono_gettext` now uses both ``msgid`` and ``msgctxt`` when present. This changes identification of translation strings in such files, breaking links to extended Weblate data such as screenshots or review states. Please ensure you commit pending changes in such files prior to upgrading. It is recommended to force-load the affected component using :djadmin:`loadpo`.
```
?

File: docs/admin/upgrade.rst [hunk 8]
```diff
@@ -1,45 +1,44 @@
 Upgrading Weblate
 =================
 
-Docker image upgrades
+Docker-image upgrades
```
Comment (nijel) [inline docs/admin/upgrade.rst:4]: Why? Nobody seems ot use it this way.

Comment (comradekingu) [reply docs/admin/upgrade.rst:4]: To distinguish between "Upgrades of the docker image", and "Image upgrades labeled "Docker"".
https://www.grammarly.com/blog/hyphen/
Used elsewhere in Weblate after the debate where I was made aware.

**Comment by comradekingu** [comment]
Somehow #6084 already existed

**Pull request closed without merging**